lib/Bio/EnsEMBL/DataCheck/Checks/CheckGeneGainLossData.pm

```diff
@@ -0,0 +1,57 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckGeneGainLossData;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CheckGeneGainLossData',
+  DESCRIPTION    => 'ncRNA and protein trees must have gene Gain/Loss trees',
```

Comment:
Is this true for the non-vert compara dbs?

Reply:
No, non-verts skip the ncRNA analyses. This test will fail for non-verts. Would you recommend I include a `skip_tests` sub? Or something else?

Reply:
It wasn't really the ncRNA I was thinking about, it was that the non-vert divisions (apart from plants) do not generate gene gain/loss trees at all. But yes, a skip_tests method is the solution, I think.